src/Build/Resources/Strings.resx

```diff
@@ -1769,13 +1769,28 @@ Utilization:          {0} Average Utilization: {1:###.0}</value>
       LOCALIZATION: {0} and {1} are file paths
     </comment>
   </data>
+  <data name="UninitializedPropertyRead" UESanitized="false" Visibility="Public">
+    <value>MSB4261: Read uninitialized property "{0}"</value>
```

Comment:
These informational messages aren't usually given error/warning codes. Do you need them?

Reply:
I'm not sure. When is the decision made to include one or note?

Reply:
As a user, if I see a code, I expect it is something I should probably fix, and not ignore. Otherwise I"m confused.

Reply:
@rainersigwald , do you agree with this? If so, I can remove the codes from the three added messages. Thanks, @danmosemsft !

Reply:
My thinking is more like "is this a straightforward status message, or something I need to search for when I see it?" but I think it mostly rolls out the same.

That policy is consistent with the codebase (not shocking, since Dan was the dev lead for MSBuild for a long time). Remove 'em :)

Reply:
K. Will do next commit! Thanks.